core/admin/EE_Admin_Page.core.php

```diff
@@ -1161,7 +1161,7 @@ public static function add_query_args_and_nonce(
                 if (strpos($key, 'nonce') !== false) {
                     continue;
                 }
-                $args[ 'wp_referer[' . $key . ']' ] = htmlspecialchars($value);
+                $args[ 'wp_referer[' . $key . ']' ] = (is_string($value) ? htmlspecialchars($value) : $value);
```

Comment:
fix is absolutely correct, but the extra brackets are unnecessary:

```suggestion
                $args[ 'wp_referer[' . $key . ']' ] = is_string($value) ? htmlspecialchars($value) : $value;
```